src/components/list-item/list-item.tsx

```diff
@@ -308,7 +307,7 @@ export class ListItem implements ConditionalSlotComponent, InteractiveComponent
             [CSS.nestedContainerHidden]: expandable ? !expanded : false
           }}
         >
-          <slot aria-description={description} onSlotchange={this.handleDefaultSlotChange} />
```

Comment:
We'll need to keep an `aria` prop in the slot, otherwise no announcement exists. Can we switch this one to `aria-label={label}`?